apps/desktop-wallet/package.json

```diff
@@ -91,7 +91,7 @@
     "concurrently": "^7.4.0",
     "cross-env": "^7.0.3",
     "dayjs": "^1.10.7",
-    "electron": "22.3.5",
```

Comment:
I would be careful upgrading electron... There was a reason I fixed the version, but I don't remember why... Let's verify that everything in the desktop wallet works, building, running dev, etc.

Reply:
Maybe that's why the CI fails